### DISCUSSION TITLE
@types/mssql not giving the correct methods and typings

### DISCUSSION THREAD

**Discussion by max-programming** [DefinitelyTyped/DefinitelyTyped#54420] [Issues with a @types package]
I installed the `mssql` npm package and then the types for mssql and now I see that it is giving me errors for the methods which exist in the library and work too. But the types are wrong or my method of using them is wrong. 
```ts
import sql from 'mssql'
import * as sql from 'mssql' // this one too doesn't work
```

![image](https://user-images.githubusercontent.com/51731966/125162856-cf619900-e1a7-11eb-879f-e5916cd620dd.png)

This is the problem 👆. Same with the `query` method.

**Comment by ghost** [comment]
🤷‍♂️ the types are apparently just not there

**Reply by max-programming** [reply]
> 🤷‍♂️ the types are apparently just not there

When I looked up to the node_modules folder I found the types and also the file was filled with the types. But they aren't appearing in the intellisense
